server/src/services/cart.service.ts

```diff
@@ -0,0 +1,30 @@
+import { ICartData } from '../interfaces/cartService.interface';
+import Cart from '../models/cart.model';
+import ItemOption from '../models/itemOption.model';
+
+class CartService {
+  private async _checkOptionValid(item_id: number, option_id: number) {
+    const targetOption = await ItemOption.findByPk(option_id, { raw: true });
+    if (targetOption?.item_id === item_id) {
+      return true;
+    }
+    return false;
```

Comment:
```suggestion
    return targetOption?.item_id === item_id;
```

전 이렇게 줄여쓰긴 하는 편인데,
이런 방식은 어떻게 생각하시나욤? 😊

Reply:
오 위 방법처럼하면 확실히 한줄에 담을 수 있겠네요 좋습니다!

Reply:
위 방법을 적용하려고 보니까 저 boolean값에따라 에러를 던져줘야 했던 부분이네요.. 저기 if문에서 바로 throw를 해버리면 될 것 같습니다 :)

Reply:
너무 길어지거나 의미를 확실히 해야할 경우에는 if 분기도 좋은 것 같습니당
사실상 주관적인 판단이기 때문에....😅
짧은 코드와 가독성 좋은 코드의 줄다리기가 항상 어려운 것 같아요ㅠㅠ